intranet/scss/components/_block-activity.scss

```diff
@@ -0,0 +1,86 @@
+.block-activity {
+  margin-bottom: rem(40px);
+
+  &__title {
+    @include opensans-semibold;
+    background-color: $green-e1;
+    margin-bottom: 0;
+    padding: rem(15px 0);
+
+    @include susy-media(mobileLarge) {
+      padding: rem(15px 30px);
+    }
+  }
+
+  &__item {
+    border-bottom: rem(1px) solid rgba(100, 143, 20, .25);
+    padding: rem(21px 0);
+
+    @include susy-media(mobileLarge) {
+      padding: rem(21px 30px);
+    }
+
+    &:hover {
+      background-color: $green-f7;
+    }
+  }
+
+  &__image {
+    float: left;
+    width: rem(40px);
+
+    img {
+      border-radius: 100%;
+      height: rem(40px);
+      width: rem(40px);
```

Comment:
add more margin-right for image. if user don't have image. It still show good. no have margin.